src/internalTable/internal-table-printer.ts

```diff
@@ -184,8 +184,15 @@ export const renderTable = (table: TableInternal): string => {
 
   renderTableHeaders(table).forEach((row) => ret.push(row));
 
-  table.rows.forEach((row) => {
+  const lastRowIndex = table.rows.length - 1;
+  table.rows.forEach((row, rowIndex) => {
     renderRow(table, row).forEach((row_) => ret.push(row_));
+    if (row.bottomBorder && rowIndex < lastRowIndex) {
```

Comment:
I would put this logic inside the renderRow part

Reply:
How about moving to another method like renderRowSeparator? I don't like the idea that renderRow would access table.rows to figure out if the current row is the last one so that it doesn't add double separator  (rowSeparator + tableBottom)

Reply:
that sounds good too